package.json

```
@@ -3,6 +3,7 @@
 	"version": "3.0.1",
 	"description": "WebdriverIO plugin for testing a Wikibase repo.",
 	"homepage": "https://github.com/wmde/wdio-wikibase",
+	"bugs": "https://phabricator.wikimedia.org/",
```

Sorry to drag this out but I always expected this to be a deep link. Arguably, `https://phabricator.wikimedia.org/tag/wikidata/` may not be an ideal tag to begin with and it certainly does not only contain topics related to this sub-component but a link to the home page does not seem to be in the best interest of the user in my eyes.

```suggestion
	"bugs": "https://phabricator.wikimedia.org/tag/wikidata/",
```

---

Yeah, I considered that, but feared that seeing ~2000 "incoming" issues might be a bit disheartening? Maybe https://phabricator.wikimedia.org/tag/browser-tests/ would be better? At the very least, I should probably add that to the READEME.md link

---

Hmm. I don't know but seeing it was done with intent makes it ok with me (for lack of a truly good option). It's an improvement over the status quo and we don't plan to have it engraved in stone.